content/admin.py

```diff
@@ -55,6 +55,13 @@ class VolunteerAboutAdmin(admin.ModelAdmin):
 class CustomPageAdmin(admin.ModelAdmin):
     list_display = ('category', 'order','title')
 
+class DonateInfoAdmin(admin.ModelAdmin):
+    fields = [ 'order', 'title', 'img', 'text']
+    list_display = ('order','title')
```

Comment:
do we need to specify this? Is this because of the list displays? Can we just do 

`admin.site.register(DonateInfo)`